### PR TITLE
Neon API handling

### PR DESCRIPTION
**Pull request by NeonKirill**
Implemented module for requesting MQ service

### CODE REVIEW

**Comment by NeonKirill** [comment]
@NeonDaniel Please firstly refer to https://github.com/NeonGeckoCom/pyklatchat/pull/4 ,yesterday testing issue was about child threads being killed while main terminated its flow

File: neon_utils/configuration_utils.py [hunk 1]
```diff
@@ -928,3 +928,11 @@ def create_config_from_setup_params(path=None) -> NGIConfig:
     # TODO: Use XDG here DM
     local_conf.write_changes()
     return local_conf
+
+
+def get_default_local_config(path=None) -> dict:
```
Comment (NeonDaniel) [inline neon_utils/configuration_utils.py:933]: Why use this vs `get_neon_local_config`? https://github.com/NeonGeckoCom/neon-skill-utils/blob/4b7b6d86b9dfd0e96fa6ab9f8fcdbe8b4bf26205/neon_utils/configuration_utils.py#L753

File: neon_utils/service_apis/__init__.py [hunk 2]
```diff
@@ -16,37 +16,103 @@
...
-    request_data = {**query_params, **{"service": str(api)}}
-    # TODO: Send request to neon_api_proxy with request_data and get a response DM
-    return {"status_code": 501,
-            "content": "Neon API not implemented",
-            "encoding": None}
+
+    response_data = dict()
+
+    try:
+
```
Comment (NeonDaniel) [inline neon_utils/service_apis/__init__.py:67]: Please remove extra newlines

File: neon_utils/service_apis/__init__.py [hunk 3]
```diff
@@ -16,37 +16,103 @@
...
+        request_data = {**query_params, **{"service": str(api)}}
+
+        LOG.debug(f'Received request data: {request_data}')
+
+        response_event = Event()
+
+        LOG.debug('Creating Neon API MQ Handler Instance...')
+
+        # TODO: decide on how to pass user configs to MQ Handler
```
Comment (NeonDaniel) [inline neon_utils/service_apis/__init__.py:76]: `get_neon_local_config` will provide default values with any user changes applied

File: tests/configuration/default_local_config.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+MQ:
```
Comment (NeonDaniel) [inline tests/configuration/default_local_config.yml:1]: Consider adding this section to `default_core_conf.yml`. In the future, we might consider moving all communication params to a separate file (Websocket, GUI websocket, MQ, etc.)

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

File: neon_utils/configuration_utils.py (hunk 1 above)
Comment (NeonKirill) [reply neon_utils/configuration_utils.py:933]: > Why use this vs `get_neon_local_config`?
> 
> https://github.com/NeonGeckoCom/neon-skill-utils/blob/4b7b6d86b9dfd0e96fa6ab9f8fcdbe8b4bf26205/neon_utils/configuration_utils.py#L753

Surely we can, just refer to the advice you gave me on Slack

**Comment by NeonDaniel** [comment]
@NeonKirill I tested this as working with the following changes:

https://github.com/NeonGeckoCom/neon_api_proxy/pull/8
https://github.com/NeonKirill/neon-skill-utils/pull/2

**Review by NeonDaniel — APPROVED** [review]

**Pull request merged**